ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -781,6 +781,45 @@ def ocs_install_verification(
         ), "Seems like MCG root secrets are public, please check"
         log.info("Noobaa root secrets are not public")
 
+    # Verify the owner of CSI deployments and daemonsets if not provider mode
+    storage_cluster_obj = get_storage_cluster()
```

Comment:
I think you need to add a condition that this code shouldn't run on a cluster client.

Reply:
Added a new condition that is commonly used in the same function.